### PR TITLE
Fixes #7111: Build the html version of the rudder man page into the manual

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7111

### CODE REVIEW

**Comment by amousset** [comment]
WIP

**Comment by amousset** [comment]
PR rebased.

File: Makefile [hunk 1]
```diff
@@ -7,6 +7,8 @@ SOURCES = $(BASENAME).txt
 TARGETS = epub html pdf readme
 DOCBOOK_DIST = xsl/xsl-ns-stylesheets
 
+RUDDER_VERSION = 3.0
```
Comment (peckpeck) [inline Makefile]: When you add a version specific information into some file, do not forget that it needs to be updated on each version change, which is hard/boring/easily forgotten ...
So you should preferably find a way to guess it automatically, or if you can't, add the information into a manual procedure.
For this one, I've seen with @VinceMacBuche and we've added it to the release procedure.

**Comment by peckpeck** [comment]
OK Good

**Pull request merged**